cpp/arcticdb/processing/operation_dispatch_unary.hpp

```diff
@@ -44,7 +44,7 @@ VariantData unary_operator(const Value& val, Func&& func) {
     details::visit_type(val.type().data_type(), [&](auto val_tag) {
         using type_info = ScalarTypeInfo<decltype(val_tag)>;
         if constexpr (!is_numeric_type(type_info::data_type)) {
-            util::raise_rte("Cannot perform arithmetic on {}", val.type());
+            util::raise_rte("Cannot perform arithmetic on {}", get_user_friendly_type_string(val.type()));
```

Comment:
No operator in exception message

Reply:
Fixed in [a907db9](https://github.com/man-group/ArcticDB/pull/1557/commits/a907db94c4366371434d048c630f796fcbc7a60e)